src/js/health-beta/health-beta-entry.jsx

```diff
@@ -1,4 +1,8 @@
 import 'core-js';
+import '../common';
+import '../../sass/rx/rx.scss';
+import '../../sass/user-profile.scss';
```

Comment:
I don't think styles need to be imported for this app.

Reply:
So I can safely remove those 3 lines?

Reply:
Yeah, I don't think those two lines are used.

Reply:
Ah I would have removed it but I'm gonna leave it to whoever maintains that just in case there are some styles being used out of there for a modal or something. I'm not familiar with that app at all.